### DISCUSSION TITLE
How to display my bottom tab through all content pages in the XF app

### DISCUSSION THREAD

**Discussion by dbeylkhanov** [MvvmCross/MvvmCross#3538] [Q&A]
I need to keep my bottom tab on all pages, not only on a Home page

My pages structure is:
- StartPage (MvxContentPage without the navbar and etc., just like splash screen), where I have the next:
 `public partial class StartPage : MvxContentPage<StartPageViewModel>`

On this page I have a link to navigate on the home page by this way:
` await _navigationService.Navigate<MainTabbedPageViewModel>();`

- MainTabbedPage with my default tabs
XAML:
```
<TabbedPage.Children>
        <pages:HomePage IconImageSource="home_icon.png" BindingContext="{Binding HomePageViewModel}">
        </pages:HomePage>
</TabbedPage.Children>
```
Behind code:
```
[MvxTabbedPagePresentation(TabbedPosition.Root, NoHistory = true, WrapInNavigationPage = true)]
    public partial class MainTabbedPage : MvxTabbedPage<MainTabbedPageViewModel>
    { ... }
```
Viewmodel:
`public class MainTabbedPageViewModel : MvxNavigationViewModel 
{...}`

- Home page 
```
[MvxTabbedPagePresentation(NoHistory = true)]
    public partial class HomePage : MvxContentPage<HomePageViewModel>
```
Viewmodel:
`public class HomePageViewModel : MvxViewModel`

And I have other MvxContentPage pages , where I want to see my bottom navbar as well because for now, my bottom nav bar is disappearing after navigating on my content pages.
e.g. Categories page,
` public partial class CategoriesPage : MvxContentPage<CategoriesPageViewModel>`
and
`public class CategoriesPageViewModel : MvxViewModel`

so, how can I do that?

p.s. 

project packages

![image](https://user-images.githubusercontent.com/7656847/64689194-8dc21300-d496-11e9-9a92-a6704dc2feb4.png)

**Comment by kharabyasser** [comment]
Did you find any solution for this one ?
